packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js

```diff
@@ -27,6 +27,25 @@ describe('plugin-conversation', () => {
       });
     });
 
+    describe('decryptComment()', () => {
+      it('calls the correct method if a array prop is passed to it', () => {
```

Comment:
1. ...`an array`...
2. Will there be other props tested in the future?

Reply:
No Matt there will not be any other props other than array for cards.